internal/resources/volumes.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 const (
-	userSecretVolName = "users-config"
+	userSecretVolName = "users-config" // #nosec
```

Comment:
Shouldn't we just disable rule [G101](https://securego.io/docs/rules/g101.html) as mentioned [here](https://github.com/securego/gosec#annotating-code)?

```suggestion
	userSecretVolName = "users-config" //#nosec G101
```

Reply:
Come to think of it if it is such a noisy rule, can we just disable G101 on the command line?

Reply:
To partially answer my own question, after looking at the gosec readme: something like `gosec -exclude=G101 ...` should work.

Reply:
ACK